src/components/Gyoithon/Gyoithon.tsx

```diff
@@ -210,141 +269,157 @@ const Gyoithon = () => {
-                        <Accordion.Panel>
-                            <Text align={"center"}>Add different ports in the same IP address once</Text>
-                            <Group grow style={{ marginTop: 10 }}>
-                                <Button onClick={ShowTarget} variant="outline">
-                                    Show Current Targets
-                                </Button>
-                                <Button onClick={ClearTarget} variant="outline">
-                                    Clear Targets
-                                </Button>
-                                <Button onClick={NextRun} variant="outline">
-                                    Next Step
-                                </Button>
-                            </Group>
-                            <form onSubmit={form.onSubmit((values) => Configure(values))}>
-                                <Group grow style={{ marginTop: 10 }}>
-                                    <TextInput label={"Target IP address"} required {...form.getInputProps("ip")} />
-                                    <TextInput label={"Target Port"} required {...form.getInputProps("port")} />
-                                    <NativeSelect
-                                        value={selectedProtocolOption}
-                                        onChange={(e) => setSelectedProtocolOption(e.target.value)}
-                                        label={"Protocol Type"}
-                                        data={protocolType}
-                                        required
-                                        placeholder={"HTTP/ HTTPS"}
-                                        {...form.getInputProps("protocol")}
-                                    />
-                                    <Button style={{ marginTop: 20 }} type={"submit"}>
-                                        ADD
-                                    </Button>
-                                </Group>
-                            </form>
-                            <form onSubmit={form.onSubmit((values) => Import(values))}>
-                                <Group grow>
-                                    <TextInput label={"Import File"} required {...form.getInputProps("import_file")} />
-                                    <Button style={{ marginTop: 20 }} type={"submit"}>
-                                        IMPORT THIS FILE
-                                    </Button>
-                                </Group>
-                            </form>
-                        </Accordion.Panel>
-                    </Accordion.Item>
-                    <form onSubmit={form.onSubmit((values) => Run({ ...values, Ml: selectedMLOption }))}>
-                        <Accordion.Item value="run">
-                            <Accordion.Control>Step 3: Run The Tool</Accordion.Control>
+        <RenderComponent
+            title={title}
+            description={description_userguide}
```

Comment:
Change to description. 
```suggestion
            description={description}
```